### PR TITLE
Update zh-TW Localization

### PR DESCRIPTION
**Pull request by auroursa**
This PR updates zh-TW localization based on the latest version.

### CODE REVIEW

**Comment by auroursa** [comment]
Currently this PR isn't quite finished. Once it's ready, I'll reopen it.

File: src/locale/locales/zh-TW/messages.po [hunk 1]
```diff
@@ -601,7 +601,7 @@ msgstr "來自 <0/>"
 
 #: src/screens/Signup/StepInfo/Policies.tsx:74
 msgid "By creating an account you agree to the {els}."
-msgstr ""
+msgstr 建立帳戶即表示你同意 {els}。""
```
Comment (surfdude29) [inline src/locale/locales/zh-TW/messages.po:604]: I realise the PR isn't finished, I just wanted to point out one tiny thing here: one of the quotation marks is in the wrong place.

Comment (auroursa) [reply src/locale/locales/zh-TW/messages.po:604]: Thanks for pointing this out, I'll fix it

**Review by pfrazee — APPROVED** [review]
Thank you auroursa!

**Pull request merged**